test/common/http/http1/conn_pool_test.cc

```diff
@@ -118,15 +120,22 @@ class ConnPoolImplForTest : public Event::TestUsingSimulatedTime, public FixedHt
   }
 
   void expectEnableUpstreamReady() {
-    EXPECT_CALL(mock_dispatcher_, post(_)).WillOnce(SaveArg<0>(&post_cb_));
+    EXPECT_FALSE(upstream_ready_enabled_);
+    EXPECT_CALL(*mock_upstream_ready_cb_, scheduleCallbackCurrentIteration())
+        .Times(1)
+        .RetiresOnSaturation();
```

Comment:
Given these mock expectations, should you consider making upstream_ready_cb a strict mock?

Reply:
will work on now, now that all h/1 and tcp tests are passing.

Reply:
done